### PR TITLE
OCM-6052 | feat: add `securityGroupIds` to the aws node pool struct

### PR DESCRIPTION
**Pull request by oriAdler**
Allow the user to provide additional security group IDs for the node pool.

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 This document describes the relevant changes between releases of the API model.
 
+## 0.0.354 Feb 12 2024
+- Add `AdditionalSecurityGroupIds` to `AWS Node Pool` type.
```
Comment (ciaranRoche) [inline CHANGES.md:6]: Can we add this to a different commit, just to preserve the history :pray:

Comment (oriAdler) [reply CHANGES.md:6]: Sure thing, split into two commits.

**Review by ciaranRoche — APPROVED** [review]
lgtm

**Pull request merged**